test/test_pcaprub_unit.rb

```diff
@@ -133,7 +133,7 @@ def test_set_datalink
   end
 
   def test_monitor
-    return if RUBY_PLATFORM =~ /mingw|win/
+    return if RUBY_PLATFORM =~ /mingw/
```

Comment:
This had a false positive on skipping this test on macOS, as the RUBY_PLATFORM was `x86_64-darwin19`